genie/genie-controller.go

```diff
@@ -324,13 +325,14 @@ func ParsePodAnnotationsForMultiIPPrefs(client *kubernetes.Clientset, k8sArgs ut
 }
 
 //  parseCNIAnnotations parses pod yaml defintion for "cni" annotations.
-func parseCNIAnnotations(annot map[string]string, client *kubernetes.Clientset, k8sArgs utils.K8sArgs) ([]string, error) {
+func parseCNIAnnotations(annot map[string]string, client *kubernetes.Clientset, k8sArgs utils.K8sArgs, conf utils.NetConf) ([]string, error) {
 	var finalAnnots []string
 
-	if len(annot) == 0 {
-		fmt.Fprintf(os.Stderr, "CNI Genie no annotations is given! Default plugin is weave! annot is %V\n", annot)
-		finalAnnots = []string{"weave"}
-	} else if strings.TrimSpace(annot["cni"]) == "" {
+	if len(annot) == 0 || strings.TrimSpace(annot["cni"]) == "" {
+		if conf.Default != "" {
```

Comment:
@erikstmartin our Slack channel is https://cni-genie.slack.com/ We feel free to post any feedback, questions and we will get back ASAP.
We can definitely have a discussion. Let me know what time works best for you (we are on Pacific time zone) and I can set up a Google Hangout or zoom meeting.